runtime/onert/core/include/backend/train/TrainableBackendContext.h

```diff
@@ -34,7 +34,7 @@ namespace train
 {
 
 using FunctionMap =
-  std::vector<std::pair<ir::OperationIndex, std::unique_ptr<exec::train::TrainableFnSequence>>>;
+  std::unordered_map<ir::OperationIndex, std::unique_ptr<exec::train::TrainableFnSequence>>;
```

Comment:
Should you modify the signature of this function? There is no problem with this itself, but it would be better to maintain the same function signature as other BackendContext codes. How about fixing this code with different backend codes in another PR?

Reply:
> Should you modify the signature of this function?

Yes, I want to change this alias declaration since `std::vector` is O(N) and `std::unordered_map` is O(1) when finding element by `ir::OperationIndex`. And this PR adds finding element int the code `auto &tn_seq = fn_map.at(op_index)`.

> There is no problem with this itself, but it would be better to maintain the same function signature as other BackendContext codes. How about fixing this code with different backend codes in another PR?

OK, I will create another PR to do it. Although there is no code to find element of the other `FunctionMap` by `ir::OperationIndex`, it would be a good idea to fix that alias declaration `onert::backend::FunctionMap` as well.